generator/mavparse.py

```diff
@@ -144,10 +145,11 @@ def __init__(self, index, description=''):
         self.description = description
 
 class MAVEnumEntry(object):
-    def __init__(self, name, value, description='', end_marker=False, autovalue=False, origin_file='', origin_line=0):
+    def __init__(self, name, value, description='', wip=False, end_marker=False, autovalue=False, origin_file='', origin_line=0):
```

Comment:
I think it makes more sense to add the `wip=false` at the end of the parameter list instead of after the description

Reply:
Ah, but you haven't considered that name, value, description AND wip are properties of the XML file and hence sensibly grouped. The others are properties of this parser. IMO doesn't make sense to shove at the end.